### ISSUE TITLE
Release v1.3.5 does not install without yarn

### ISSUE THREAD

**Issue by kevinbosman** [chinchiheather/tslint-lines-between-class-members#28]
Hi,

The 1.3.5 release on npm won't install on a standard node installation. The `postinstall` script tries to do

```
cd ./integration && yarn
```
which results in:
```
The system cannot find the path specified.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tslint-lines-between-class-members@1.3.5 postinstall: `cd ./integration && yarn`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tslint-lines-between-class-members@1.3.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
````
Surely doing anything with yarn should not be required on postinstall?

**Comment by rallets** [comment]
same problem for me. 1.3.4 works fine.

**Comment by valera33** [comment]
even if yarn is installed on the system, same error:
```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! tslint-lines-between-class-members@1.3.5 postinstall: `cd ./integration && yarn`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the tslint-lines-between-class-members@1.3.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/valeri/.npm/_logs/2019-07-20T21_51_56_905Z-debug.log
valeri@foo (foo*) ~/workspace/backup: yarn --version
1.17.3
```

**Comment by chinchiheather** [comment]
Sorry about this, I have released a new version 1.3.6 which should install for you and deprecated the 1.3.5 version.

It is interesting that 1.3.4 installed though as that is the version where the postinstall script was added

**Comment by rallets** [comment]
Thanks, now it works
